### PR TITLE
Minor fixes for hook-me.sh script

### PR DESCRIPTION
**Pull request by prashanth26**
**How to categorize this PR?**

/area dev-productivity
/kind cleanup
/priority normal

**What this PR does / why we need it**:
The hook-me.sh had a minor type where two parameters in the function were swtiched. This PR fixes it. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement developer
Minor fixes for hook-me.sh script
```

### CODE REVIEW

**Review by zanetworker — DISMISSED** [review]
/lgtm

File: extensions/hack/hook-me.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ waitForInletsLBToBeReady(){
       do
         sleep 2s
       done
-      echo $(kubectl -n $namespace get svc inlets-lb -o go-template="{{ index (index  .status.loadBalancer.ingress 0).hostname }}")
+      echo $(kubectl -n $namespace get svc inlets-lb -o go-template="{{ index (index  .status.loadBalancer.ingress 0).hostname }}")
```
Comment (rfranzke) [inline extensions/hack/hook-me.sh]: trailing whitespace

Comment (prashanth26) [reply extensions/hack/hook-me.sh]: Done.  Refer - https://github.com/gardener/gardener/pull/2473/commits/95dce3ec57fd4509fc195bd2d8fdc428478607d6.

**Comment by rfranzke** [comment]
Can you please squash the commits?

**Comment by prashanth26** [comment]
Done.

**Review by rfranzke — APPROVED** [review]

**Pull request merged**